yamljs/yamljs.d.ts

```diff
@@ -1,9 +1,9 @@
-// Type definitions for yamljs 0.2.1
+// Type definitions for yamljs 0.2.3
 // Project: https://github.com/jeremyfa/yaml.js
 // Definitions by: Tim Jonischkat <http://www.tim-jonischkat.de>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
-declare module "yamljs" {
```

Comment:
Thanks for the response. Can you please help me understand why the module
needs to be declared as "yamljs" when the namesapce is window.YAML? The
current version is declared this way but does not work. I'm new-ish to
typescript so i'd like to understand the reasoning.

On Mon, Aug 3, 2015 at 8:13 AM, Masahiro Wakame <notifications@github.com>
wrote:

> In yamljs/yamljs.d.ts
> <https://github.com/borisyankov/DefinitelyTyped/pull/5167#discussion_r36088488>
> :
>
> >  // Project: https://github.com/jeremyfa/yaml.js
> >  // Definitions by: Tim Jonischkat <http://www.tim-jonischkat.de>
> >  // Definitions: https://github.com/borisyankov/DefinitelyTyped
> >
> > -declare module "yamljs" {
>
> please keep this line. use this pattern
> https://github.com/borisyankov/DefinitelyTyped/blob/master/jszip/jszip.d.ts#L189-L191
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/borisyankov/DefinitelyTyped/pull/5167/files#r36088488>
> .
>



-- 
J.